requirements/_base.in

```diff
@@ -3,13 +3,12 @@
 #
 
 aiohttp
-openapi-core
 docker
-psycopg2-binary
 ptvsd
 semantic_version
 tenacity
 trafaret_config
+werkzeug
```

Comment:
for a next feature request: we should replace trafaret by pydantic. it's much easier to use.